common/libs/validator.js

```diff
@@ -0,0 +1,16 @@
+import WalletAddressValidator from 'wallet-address-validator';
+import ethUtil from 'ethereumjs-util';
+
+export default class Validator {
+  isValidETHAddress = function(address) {
```

Comment:
if !address return false;

if address === '0x...' return false;

return check();

would be easier to read